### PR TITLE
Add summaries main (day 1)

### PR DESCRIPTION
**Pull request by colleenjg**
Added summaries of the main conference for day 1.

### CODE REVIEW

**Comment by colleenjg** [comment]
Partially addresses issue #2. Day 2 still to be drafted.

File: paper.md [hunk 1]
```diff
@@ -14,27 +14,38 @@ Probably leave this for last, if we need it.
...
...ompromise between optimizing temporal resolution, spatial resolution and brain coverage. Fortunately, new technologies continue to push the boundaries of this trade-off, improving the combined resolution and coverage that can be achieved using a single recoding modality. These include the Neuropixel, a silicon probe which allows high-density simultaneous recording of hundreds of neurons in awake and freely moving animals [Jun et al., 2017, Nature]. More recently, thin-film, multithousand-channel recording grids using platinum nanrods (PtNRGrids) have been developed which enable highly dense recordings of brain surface activity in human patients, allowing researchers to resolve the dynamics of normal and pathological activity at high spatial and temporal resolutions [Tchoe et al., 2022, Science Translational Medicine]. In the field of two-photon imaging, light beads microscopy now enables volumetric recordings of single cell activity across the entire cortex [Demas et al., 2021, Nature Methods]. At the circuit level, improvements in voltage imaging now allow researchers to optically resolve sub-threshold activity across the dendritic tree of single cells [Adam et al., 2019, Nature].
```
Comment (CodyCBakerPhD) [inline paper.md:21]:
```suggestion
When selecting brain recording technologies to use in their studies, researchers generally must make a compromise between optimizing temporal resolution, spatial resolution and brain coverage. Fortunately, new technologies continue to push the boundaries of this trade-off, improving the combined resolution and coverage that can be achieved using a single recoding modality. These include Neuropixels probes, a silicon probe which allows high-density simultaneous recording of hundreds of neurons in awake and freely moving animals [Jun et al., 2017, Nature]. More recently, thin-film, multithousand-channel recording grids using platinum nanrods (PtNRGrids) have been developed which enable highly dense recordings of brain surface activity in human patients, allowing researchers to resolve the dynamics of normal and pathological activity at high spatial and temporal resolutions [Tchoe et al., 2022, Science Translational Medicine]. In the field of two-photon imaging, light beads microscopy now enables volumetric recordings of single cell activity across the entire cortex [Demas et al., 2021, Nature Methods]. At the circuit level, improvements in voltage imaging now allow researchers to optically resolve sub-threshold activity across the dendritic tree of single cells [Adam et al., 2019, Nature].
```

Seems the 'official' brand name is "Neuropixels"

Comment (CodyCBakerPhD) [inline paper.md:21]:
```suggestion
When selecting brain recording technologies to use in their studies, researchers generally must make a compromise between optimizing temporal resolution, spatial resolution and brain coverage. Fortunately, new technologies continue to push the boundaries of this trade-off, improving the combined resolution and coverage that can be achieved using a single recording modality. These include the Neuropixel, a silicon probe which allows high-density simultaneous recording of hundreds of neurons in awake and freely moving animals [Jun et al., 2017, Nature]. More recently, thin-film, multithousand-channel recording grids using platinum nanrods (PtNRGrids) have been developed which enable highly dense recordings of brain surface activity in human patients, allowing researchers to resolve the dynamics of normal and pathological activity at high spatial and temporal resolutions [Tchoe et al., 2022, Science Translational Medicine]. In the field of two-photon imaging, light beads microscopy now enables volumetric recordings of single cell activity across the entire cortex [Demas et al., 2021, Nature Methods]. At the circuit level, improvements in voltage imaging now allow researchers to optically resolve sub-threshold activity across the dendritic tree of single cells [Adam et al., 2019, Nature].
```

Comment (CodyCBakerPhD) [reply paper.md:21]: Hmm.. normally the changelog displays highlighted changes...

This was a typo: 'recoding' -> 'recording' in first sentence

File: paper.md [hunk 2]
```diff
@@ -14,27 +14,38 @@ Probably leave this for last, if we need it.
...
...earch laboratories, increasing the technological skills required to collect, manage and analyse this data. Efficient, scalable and reproducible research tools are needed to support researchers in collecting high-quality datasets and performing reliable analyses. To this end, new commercial services are emerging, like DataJoint, which helps laboratories design and scale end-to-end data collection, analysis and sharing pipelines [Yatsenko et al., 2018, arXiv], and CatalystNeuro which works with neuroscientists to provide customized software solutions to their specific needs. Finally, open web-based tools are also being developed to allow researchers to interact with and visualize the increasingly accessible, complex and plentiful data available online. Neurosift allows users to visualize data recorded in NWB files hosted in the cloud, without having to download large datasets locally, while Protocass is currently being developed as a web app for analyzing data stored in the cloud or locally. As figures in research papers become increasingly complex and multi-dimensional, Figurl provides a framework for creating and sharing interactive visualizations of data and data analysis results.
```
Comment (CodyCBakerPhD) [inline paper.md:29]:
```suggestion
The large amounts of data being collected and their growing complexity also place high demands on research laboratories, increasing the technological skills required to collect, manage and analyse this data. Efficient, scalable and reproducible research tools are needed to support researchers in collecting high-quality datasets and performing reliable analyses. To this end, new commercial services are emerging, like DataJoint, which helps laboratories design and scale end-to-end data collection, analysis and sharing pipelines [Yatsenko et al., 2018, arXiv], and CatalystNeuro which works with neuroscientists to provide customized software solutions to their specific needs. Finally, open web-based tools are also being developed to allow researchers to interact with and visualize the increasingly accessible, complex and plentiful data available online. Neurosift allows users to visualize data recorded in NWB files hosted in the cloud, without having to download large datasets locally, while Dendro is currently being developed as a web app for analyzing data stored in the cloud or locally. As figures in research papers become increasingly complex and multi-dimensional, Figurl provides a framework for creating and sharing interactive visualizations of data and data analysis results. 
```

"Protocass" has now evolved into "Dendro". Will try to get something citable for this

File: paper.md [hunk 3]
```diff
@@ -14,27 +14,38 @@ Probably leave this for last, if we need it.
...
+initiatives like these, which connect traditional academic research laboratories around the world and generate foundational databases for the field, will become increasingly important.
```
Comment (CodyCBakerPhD) [inline paper.md:39]:
```suggestion
initiatives like these, academic research laboratories around the world and generate foundational databases for the field, will become increasingly important.
```

just testing something with GitHub with this comment

File: paper.md [hunk 4]
```diff
@@ -14,27 +14,38 @@ Probably leave this for last, if we need it.
...
...urces to collect them in their own laboratories. Experimental proposals from external scientists are selected yearly for their feasibility and scientific merit, and deployed in close collaboration, using the Allen Brain Observatory pipeline. After an initial period during which the proposing team has priority access, the datasets are published for open access use by the neuroscience community at large. The International Brain Laboratory (IBL), for its part, provides a decentralized model for ensuring the reproducibility and reliability of neuroscience data and research. For its first project, the IBL, comprising researchers from 22 laboratories in 6 countries, collaborated on designing a standardized experiment for probing the role of neural circuits in decision-making. Once the experiment, standardized from data collection to analysis, had been carried out in each laboratory, the findings and data were jointly published, to ensure maximal reproducibility and reliability of the findings [The International Brain Laboratory et al., 2021, eLife; The International Brain Laboratory et al., 2023, bioRxiv]. As of neuroscience research matures and projects increase in scope and complexity,
```
Comment (CodyCBakerPhD) [inline paper.md:38]:
```suggestion
Also leveraging the Allen Institute's extensive resource, the OpenScope initiative was created to support researchers who have ideas for valuable datasets, but lack the necessary resources to collect them in their own laboratories. Experimental proposals from external scientists are selected yearly for their feasibility and scientific merit, and deployed in close collaboration, using the Allen Brain Observatory pipeline. After an initial period during which the proposing team has priority access, the datasets are published for open access use by the neuroscience community at large. The International Brain Laboratory (IBL), for its part, provides a decentralized model for ensuring the reproducibility and reliability of neuroscience data and research. For its first project, the IBL, comprising researchers from 22 laboratories in 6 countries, collaborated on designing a standardized experiment for probing the role of neural circuits in decision-making. Once the experiment, standardized from data collection to analysis, had been carried out in each laboratory, the findings and data were jointly published, to ensure maximal reproducibility and reliability of the findings [The International Brain Laboratory et al., 2021, eLife; The International Brain Laboratory et al., 2023, bioRxiv]. Ae research matures and projects increase in scope and complexity,
```

Just testing something with GitHub with this comment

File: paper.md (hunk 3 above)
Comment (CodyCBakerPhD) [reply paper.md:39]: Aha, yes. I see now. Highlighting only occurs for shorter lines (long paragraphs on a single line don't seem to trigger it)

**Comment by CodyCBakerPhD** [comment]
@colleenjg All in all I think these look great! Sorry for taking so long to get around to reviewing

Corrected one typo and the official name for one of the devices

Looking forward to part 2!

By the way, how are you tracking the bibliography for each of the references? It seems like you did quite a bit of digging to find all those

File: paper.md [hunk 5]
```diff
@@ -14,27 +14,38 @@ Probably leave this for last, if we need it.
...
...mpromise between optimizing temporal resolution, spatial resolution and brain coverage. Fortunately, new technologies continue to push the boundaries of this trade-off, improving the combined resolution and coverage that can be achieved using a single recording modality. These include the Neuropixel, a silicon probe which allows high-density simultaneous recording of hundreds of neurons in awake and freely moving animals [Jun et al., 2017, Nature]. More recently, thin-film, multithousand-channel recording grids using platinum nanrods (PtNRGrids) have been developed which enable highly dense recordings of brain surface activity in human patients, allowing researchers to resolve the dynamics of normal and pathological activity at high spatial and temporal resolutions [Tchoe et al., 2022, Science Translational Medicine]. In the field of two-photon imaging, light beads microscopy now enables volumetric recordings of single cell activity across the entire cortex [Demas et al., 2021, Nature Methods]. At the circuit level, improvements in voltage imaging now allow researchers to optically resolve sub-threshold activity across the dendritic tree of single cells [Adam et al., 2019, Nature].
```
Comment (CodyCBakerPhD) [inline paper.md:21]:
```suggestion
When selecting brain recording technologies to use in their studies, researchers generally must make a compromise between optimizing temporal resolution, spatial resolution and brain coverage. Fortunately, new technologies continue to push the boundaries of this trade-off, improving the combined resolution and coverage that can be achieved using a single recording modality. These include Neuropixels, a silicon probe which allows high-density simultaneous recording of hundreds of neurons in awake and freely moving animals [Jun et al., 2017, Nature]. More recently, thin-film, multithousand-channel recording grids using platinum nanorods (PtNRGrids) have been developed which enable highly dense recordings of brain surface activity in human patients, allowing researchers to resolve the dynamics of normal and pathological activity at high spatial and temporal resolutions [Tchoe et al., 2022, Science Translational Medicine]. In the field of two-photon imaging, light beads microscopy now enables volumetric recordings of single cell activity across the entire cortex [Demas et al., 2021, Nature Methods]. At the circuit level, improvements in voltage imaging now allow researchers to optically resolve sub-threshold activity across the dendritic tree of single cells [Adam et al., 2019, Nature].
```

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**